PhysicsTools/IsolationAlgos/BuildFile.xml

```diff
@@ -8,7 +8,6 @@
 <use name="FWCore/PluginManager"/>
 <use name="FWCore/ParameterSet"/>
 <use name="DataFormats/RecoCandidate"/>
-<use name="FWCore/MessageLogger"/>
```

Comment:
this is included in e.g. plugins/CandIsoDepositProducer.cc and PhysicsTools/IsolationAlgos/plugins does not have FWCore/MessageLogger.
Shouldn't it be added there?

I vaguely recall that a similar case occurred before; I thought that this was taken care of already in the script.
(one example below in RecoTracker/FinalTrackSelectors/ vs plugins/ shows that this is handled in some way)

Reply:
Hi! No, the script doesn't consider this because I don't think we came to any clear conclusion what to do in these cases. I though if it still compiles it's fine. But okay, I will always add the removed dependency in `plugins` if necessary from now.

I handled the case in `RecoTracker/FinalTrackSelectors` manually, as this was mentioned in a comment by @davidlange6 in another commit.